RELEASE.md

```diff
@@ -8,6 +8,7 @@ This is a  document to describe the release process for the verifier.
 - [Verify provenance](#verify-provenance)
 - [Update documentation](#update-documentation)
 - [Update builders](#update-builders)
+- [Add tests for new builder release version](#update-tests)
```

Comment:
should this not be in the builder's RELEASE.md? We already have it mentioned, but you can add the link to the workflows u created

Reply:
moving it!